### PR TITLE
Second label

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: src/components/Node/Node.js [hunk 1]
```diff
@@ -290,6 +290,7 @@ export default memo(function Node({
           variant={"link"}
           disabled={node.loadingSiblings}
           onClick={() => toggleSiblings(node)}
+          title={(node._spousesExpanded ? "Collapse" : "Expand") + " siblings"}
```
Comment (ogroppo) [inline src/components/Node/Node.js:293]: _spousesExpanded => _siblingsExpanded

File: src/components/Node/Node.js [hunk 2]
```diff
@@ -414,6 +415,7 @@ const ThemedContent = styled.div`
   ${({ theme }) => theme.nodeFlexDirection === "row" && `padding-left: 4px`};
   ${({ theme }) => theme.nodeFlexDirection === "column" && `padding-top: 4px`};
   .label {
+    // word-break: break-all;
```
Comment (ogroppo) [inline src/components/Node/Node.js:418]: what about long names?

File: src/constants/langs.js [hunk 3]
```diff
@@ -1,3 +1,6 @@
+import { Dropdown } from "react-bootstrap";
+import React from "react";
+
```
Comment (ogroppo) [inline src/constants/langs.js:3]: not needed...

File: src/constants/langs.js [hunk 4]
```diff
@@ -445,3 +448,13 @@ export const LANGS = Object.keys(LANG_MAP).map((code) => {
 
 export const DEFAULT_LANG = LANGS.find(({ code }) => code === "en");
 export const DEFAULT_LANGS_CODES = ["en", "fr", "de", "it", "es", "nl", "pt"];
+export const SECOND_LABELS = [
+  {
+    code: "nickname",
```
Comment (ogroppo) [inline src/constants/langs.js:453]: NICKNAME_ID

File: src/lib/addSecondLangLabel.js [hunk 5]
```diff
@@ -1,7 +1,21 @@
+import { BIRTH_NAME_ID, NICKNAME_ID } from "../constants/properties";
+
 export default function addSecondLangLabel(entity, languageCode) {
   if (!languageCode) return;
-  const { labels } = entity;
+  const { labels, simpleClaims } = entity;
   if (!labels) return;
+  if (languageCode === "nickname") {
```
Comment (ogroppo) [inline src/lib/addSecondLangLabel.js:7]: === NICKNAME_ID

File: src/lib/addSecondLangLabel.js [hunk 6]
```diff
@@ -1,7 +1,21 @@
+import { BIRTH_NAME_ID, NICKNAME_ID } from "../constants/properties";
+
 export default function addSecondLangLabel(entity, languageCode) {
```
Comment (ogroppo) [inline src/lib/addSecondLangLabel.js:3]: addSecondLangLabel =>  addSecondLabel, languageCode => propCode

**Review by ogroppo — CHANGES_REQUESTED** [review]

File: src/components/Node/Node.js (hunk 1 above)
Comment (mshd) [reply src/components/Node/Node.js:293]: Oh right

File: src/components/Node/Node.js (hunk 2 above)
Comment (mshd) [reply src/components/Node/Node.js:418]: Yes, don't know, I don't like word-break as it will break randomly

File: src/constants/langs.js (hunk 3 above)
Comment (mshd) [reply src/constants/langs.js:3]: sorry was automatic

File: src/constants/langs.js (hunk 4 above)
Comment (mshd) [reply src/constants/langs.js:453]: Yeah sure

File: public/index.html [hunk 7]
```diff
@@ -15,6 +15,11 @@
     <meta name="twitter:title" content="$OG_TITLE" />
     <meta name="twitter:description" content="$OG_DESCRIPTION" />
     <meta name="twitter:image" content="$OG_IMAGE" />
+    <script
+      async
+      src="https://platform.twitter.com/widgets.js"
+      charset="utf-8"
+    ></script>
```
Comment (ogroppo) [inline public/index.html:22]: https://dev.to/heymarkkop/embed-twitter-widget-on-reactjs-1768
you can probably use this package https://www.npmjs.com/package/react-twitter-embed

File: src/components/Node/Node.js [hunk 8]
```diff
@@ -105,10 +105,10 @@ export default memo(function Node({
       ? () => setThumbnailIndex((thumbnailIndex + 1) % thumbnails.length)
       : null;
 
-  const hasSecondLang =
-    secondLang &&
-    node.data.secondLangLabel &&
-    node.data.label !== node.data.secondLangLabel;
+  const hassecondLabel =
```
Comment (ogroppo) [inline src/components/Node/Node.js:108]: minor: hassecondLabel => hasSecondLabel

File: src/components/Node/Node.js (hunk 2 above)
Comment (ogroppo) [reply src/components/Node/Node.js:418]: would you rather have a name that gets out of the node bounds?

File: src/layout/Footer/Footer.js [hunk 9]
```diff
@@ -23,6 +23,26 @@ export default function Footer() {
...
+            <a
+              href={`https://twitter.com/EntitreeApp?ref_src=twsrc%5Etfw`}
+              className="twitter-follow-button"
+              // data-size="large"
+              // data-show-screen-name=""
+              // data-show-count=""
+            >
+              Follow EntiTree
+            </a>
+          </Nav.Item>
```
Comment (ogroppo) [inline src/layout/Footer/Footer.js:45]: Hopefully the \<TwitterTimelineEmbed/\> will work for you

File: src/layout/Footer/Footer.scss [hunk 10]
```diff
@@ -6,6 +6,15 @@
       padding-right: 0.5rem;
       padding-left: 0.5rem;
     }
+    .nav-item {
+      margin-top: 12px;
+      margin-left: 5px;
+    }
+    @media (max-width: 600px) {
+      #twitterFollowButton {
+        display: none;
```
Comment (ogroppo) [inline src/layout/Footer/Footer.scss:15]: no follow on mobile?

**Review by ogroppo — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 7 above)
Comment (mshd) [reply public/index.html:22]: I was trying to avoid using a package, but I can use it if you like

File: src/components/Node/Node.js (hunk 2 above)
Comment (mshd) [reply src/components/Node/Node.js:418]: Yes

File: src/lib/addSecondLangLabel.js (hunk 5 above)
Comment (mshd) [reply src/lib/addSecondLangLabel.js]: Ok

File: src/lib/addSecondLangLabel.js (hunk 6 above)
Comment (mshd) [reply src/lib/addSecondLangLabel.js]: Ok

File: src/layout/Footer/Footer.js (hunk 9 above)
Comment (mshd) [reply src/layout/Footer/Footer.js:45]: what's that?

File: src/layout/Footer/Footer.scss (hunk 10 above)
Comment (mshd) [reply src/layout/Footer/Footer.scss:15]: No space dude

Comment (mshd) [reply src/layout/Footer/Footer.scss:15]: better have a popup after 5

**Pull request merged**